browser/src/server.ts

```diff
@@ -25,6 +26,28 @@ async function main (): Promise<void> {
     res.send('OK')
   })
 
+  // eslint-disable-next-line @typescript-eslint/no-misused-promises
+  app.post('/download', async (req: Request, res: Response) => {
```

Comment:
No need to start a browser session for this since we're using axios to get the content.

Reply:
I think the problem is that you won't be able to get content behind javascript

Reply:
I think this will primarily be used for downloading files and content from non-web pages. Maybe that's not true.